gdal/ci/travis/mingw_w64/script.sh

```diff
@@ -7,8 +7,8 @@ cd gdal
 wine64 apps/gdalinfo.exe --version
 cd ../autotest
 # Does not work under wine
-rm gcore/gdal_api_proxy.py
-rm gcore/rfc30.py
+rm gcore/gdal_api_proxy.py gcore/rfc30.py
+rm gnm/gnm_test.py
```

Comment:
should these be `@skip`ed instead? Feels like messing with the test files in obscure CI bash scripts will make it harder to run these tests locally.

Reply:
Probably. Not sure about the gcore ones, but in the gnm case it'll need to be ignored with `--ignore` because the file doesn't even import. I'll see what I can do with this

Reply:
Unless this is trivial changes (but even trivial tend to be time consuming when this is things on a CI config you don't have locally), I'd suggest to collect those 'nice-to-improve' things in some notes for a later ticket, and concentrate on what is strictly needed to make all CI happy.